src/libtriton/bindings/python/objects/pyTritonContext.cpp

```diff
@@ -677,6 +622,9 @@ namespace triton {
               return PyErr_Format(PyExc_TypeError, "Callbacks::addCallback(): Invalid kind of callback.");
           }
         }
+        catch (const triton::exceptions::PyCallbacks&) {
+          return nullptr;
+        }
```

Comment:
Can you explain this?

Reply:
see above comment.

Reply:
Got it but still have another question :). Just to be sure, do you think we have to catch `PyCallback` also in others pyobjects?

Reply:
You mean in more places than I have added it or less ?
more: The PyCallbacks exception message is meant to inform of a missing catch: the catch blocks on general c++ exception I saw everywhere do raise a python error with the exception string.
less: Some catch blocks may be unnecessary, I initially put too much and ended up trimming the commit when composing the branch. Some indirect calls are less obvious than others, for instance convertRegisterToSymbolicVariable do call the simplification since it uses newSymbolicExpression.
If it needs to be as lean as possible, I'll check the whole list and add a comment with the call chain involved at each catch location.